### DISCUSSION TITLE
iphone only build

### DISCUSSION THREAD

**Discussion by horiaradu** [fastlane/fastlane#17586] [Q&A] [answered]
Hi,

I have an iphone only app. When I upload to the app store, the app appears to be available for ipads as well.
How can I restrict this?

Thanks!

**Comment by rogerluan** [comment, marked as answer]
Hey @horiaradu 👋 

I couldn't find official documentation to point you to, but: iPad users **must** (and will always) be able to download iPhone-only apps and execute them in compatibility/emulation mode. So if your app doesn't officially support iPads (as per your project configurations), those iPad users will be forced to use your app under a very ugly "compatibility mode", in which the screen is stretched to pretend it's a phone.

I recommend simply adding support to iPad, even if it's the bare minimum, it will look better than that compatibility mode 😬

**Reply by horiaradu** [reply]
awesome! thanks a lot for the reply!
